src/puppetlabs/puppetdb/client.clj

```diff
@@ -2,7 +2,7 @@
   (:require [puppetlabs.puppetdb.http :as http]
             [clojure.tools.logging :as log]
             [puppetlabs.puppetdb.reports :as reports]
-            [clj-http.client :as http-client]
+            [puppetlabs.http.client.sync :as http-client]
```

Comment:
Looks like this might be aliased as both http-client and pl-http -- I wondered, if we're switching to primarily/only use this lib now, we might be able to just call it `http`, i.e. `http/get`, etc. or use some other fairly friendly, short name, while we're making changes.